index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
You have successfully added a style sheet to your project :ok_hand:, but actually, you never used it :stuck_out_tongue_winking_eye:, CSS makes the front end of a website shine :sunglasses: and it creates a great user experience, u can use it also to arrange the parts of the website.
More information here :point_right: [https://www.devopsschool.com/blog/what-is-css-and-its-importance-in-web-development/](url).

Reply:
- [ ] Good job ... Your review is awesome :clap: But please make sure to always phrase the comments politely, as suggestions instead of orders. For example: "Please adjust margin to 5px for label element because that would improve the page more accessibility ..." :+1:

Reply:
You have successfully added a style sheet to your project 👌, please make sure to CSS, CSS makes the front end of a website shine 😎 and it creates a great user experience, u can use it also to arrange the parts of the website.
More information here 👉 [https://www.devopsschool.com/blog/what-is-css-and-its-importance-in-web-development/](https://github.com/Abdelaziz-Ali/Review-for-SurveyFormHTMLproject/pull/1/files/url).